specifications/api/swagger.yaml

```diff
@@ -4025,7 +4027,10 @@ paths:
       tags: ["Migrations"]
       summary: Migrate
       description: |
-        <p align="right">status: <strong>in development</strong></p>
+        <p align="right">status: <strong>disabled</strong></p>
+        <strong>⚠️IMPORTANT⚠️</strong> This endpoint has been temporarily disabled with the introduction of multi-assets UTxO. It'll be enabled again soon.
```

Comment:
```suggestion
        <strong>⚠️IMPORTANT⚠️</strong> This endpoint has been temporarily disabled with the introduction of multi-assets UTxO. It will be enabled again soon.
```
"It'll" sounds a bit weird in this setting to me (also applies to the other uses)

Reply:
https://github.com/input-output-hk/cardano-wallet/compare/1809e2ce9c7bc2df4d71fb0a049d50df550e839f..54d85d7e0c142dbdd285f80be3ba51ec446b115b